.env.sample

```
@@ -0,0 +1,22 @@
+# ports
+WEB_SERVER_PORT=3000
+SSH_SERVER_PORT=2222
+
+# your domain
+FQDN=your.domain.com
+NEXTAUTH_URL=https://your.domain.com
+
+# secrets
+NEXTAUTH_SECRET=your-secret
+CRONJOB_KEY=your-other-secret
+
+# UID:GID must match the user and group ID of the host folders and must be > 1000
+UID=1001
+GID=1001
```

```suggestion
UID=$USER
GID=$USER
```

Could also use `$USER` here, which would make it runnable more out-of-the-box?